sql/item_mods.sql

```diff
@@ -21955,56 +21961,56 @@ INSERT INTO `item_mods` VALUES (18281,356,57); -- Stage 4 Ragnarok - Scourge
 INSERT INTO `item_mods` VALUES (18283,25,25); -- Ragnarok LV80 - ACC +25
 INSERT INTO `item_mods` VALUES (18283,165,7); -- Crit Rate +7%
 INSERT INTO `item_mods` VALUES (18283,355,57); -- Scourge
-INSERT INTO `item_mods` VALUES (18283,506,50); -- Extra DMG Chance 5%
+INSERT INTO `item_mods` VALUES (18283,506,70); -- Extra DMG Chance 7%
 INSERT INTO `item_mods` VALUES (18283,507,250); -- Extra DMG X2.5
 INSERT INTO `item_mods` VALUES (18287,356,73); -- Stage 4 Guttler - Onslaught
 INSERT INTO `item_mods` VALUES (18288,23,20); -- Guttler LV75 - ATT +20
 INSERT INTO `item_mods` VALUES (18288,355,73); -- Onslaught
 INSERT INTO `item_mods` VALUES (18288,431,1); -- Additional Effect - scripts\globals\items\guttler.lua
-INSERT INTO `item_mods` VALUES (18288,506,50); -- Extra DMG Chance 5%
+INSERT INTO `item_mods` VALUES (18288,506,70); -- Extra DMG Chance 7%
 INSERT INTO `item_mods` VALUES (18288,507,250); -- Extra DMG X2.5
 INSERT INTO `item_mods` VALUES (18289,23,25); -- Guttler LV80 - ATT +25
 INSERT INTO `item_mods` VALUES (18289,355,73); -- Onslaught
 INSERT INTO `item_mods` VALUES (18289,431,1); -- Additional Effect - scripts\globals\items\guttler.lua
-INSERT INTO `item_mods` VALUES (18289,506,50); -- Extra DMG Chance 5%
+INSERT INTO `item_mods` VALUES (18289,506,70); -- Extra DMG Chance 7%
 INSERT INTO `item_mods` VALUES (18289,507,250); -- Extra DMG X2.5
 INSERT INTO `item_mods` VALUES (18293,356,89); -- Stage 4 Bravura - Metatron Torment
 INSERT INTO `item_mods` VALUES (18294,25,20); -- Bravura LV75 - ACC +20
 INSERT INTO `item_mods` VALUES (18294,355,89); -- Metatron Torment
 INSERT INTO `item_mods` VALUES (18294,431,1); -- Additional Effect - scripts\globals\items\bravura.lua
-INSERT INTO `item_mods` VALUES (18294,506,50); -- Extra DMG Chance 5%
+INSERT INTO `item_mods` VALUES (18294,506,80); -- Extra DMG Chance 8%
 INSERT INTO `item_mods` VALUES (18294,507,200); -- Extra DMG X2
 INSERT INTO `item_mods` VALUES (18295,25,25); -- Bravura LV80 - ACC +25
 INSERT INTO `item_mods` VALUES (18295,355,89); -- Metatron Torment
 INSERT INTO `item_mods` VALUES (18295,431,1); -- Additional Effect - scripts\globals\items\bravura.lua
-INSERT INTO `item_mods` VALUES (18295,506,50); -- Extra DMG Chance 5%
+INSERT INTO `item_mods` VALUES (18295,506,80); -- Extra DMG Chance 8%
 INSERT INTO `item_mods` VALUES (18295,507,200); -- Extra DMG X2
 INSERT INTO `item_mods` VALUES (18299,356,121); -- Stage 4 Gugnir - Geirskogul
 INSERT INTO `item_mods` VALUES (18300,25,20); -- Gungnir LV75 - ACC +20
 INSERT INTO `item_mods` VALUES (18300,355,121); -- Geirskogul
 INSERT INTO `item_mods` VALUES (18300,431,1); -- Additional Effect - scripts\globals\items\gungnir.lua
-INSERT INTO `item_mods` VALUES (18300,506,50); -- Extra DMG Chance %5
-INSERT INTO `item_mods` VALUES (18300,507,200); -- Extra DMG X2
+INSERT INTO `item_mods` VALUES (18300,506,70); -- Extra DMG Chance 7%
+INSERT INTO `item_mods` VALUES (18300,507,250); -- Extra DMG X2.5
```

Comment:
Seems like it would be something better under latent, rather than item mods, since you can build in the situational in latents. But that is just my two cents worth.